pick.lic

```diff
@@ -79,15 +79,26 @@ class LockPicker
         next
       end
 
-      until 'not even locked' == bput("pick my #{box} blind", 'not even locked', 'Roundtime')
-        waitrt?
-        next unless DRSkill.getxp('Locksmithing') >= 30
```

Comment:
Whoops.  Looks like it will finish picking the current box even if locked.  Fix pushed